internal/testplanet/planet.go

```diff
@@ -69,6 +70,7 @@ type Config struct {
 	StorageNodeCount int
 	UplinkCount      int
 
+	RS          *uplink.RSConfig
```

Comment:
This doesn't make sense as a testplanet config. RS is a configuration about how you upload a single object / bucket files, and not a global thing. This probably makes more sense as an argument or modifier to uplink.

The rational is that tests should upload with different configurations.

Reply:
Maybe make the argument as an option to upload?

Reply:
One approach would be to not make it part of the config, but instead just allow overriding the uplink config itself and defaulting as usual.

Reply:
Do we want to pass in the entire uplink config, or just the uplink RSConfig?

Reply:
The problem is that the config is "per bucket". At the moment I would pass in minimal, since once we flesh out more of the uplink api, we can start removing the test code from here.